package.json

```diff
@@ -41,6 +41,7 @@
     "next-compose-plugins": "^2.2.1",
     "next-transpile-modules": "^10.0.0",
     "prettier": "^2.7.1",
+    "puppeteer": "^19.10.1",
```

Comment:
Can we use the playwright (https://www.npmjs.com/package/playwright) here? We've just moved from puppeeter to playwright in other projects)

Reply:
94fcdba80ad7e467d8ecc2e4a50e131d84e0fa25